OWNERS

```diff
@@ -5,6 +5,3 @@ approvers:
 
 reviewers:
 - kubernetes-csi-reviewers
-
-emeritus_reviewers:
-- davidz627
```

Comment:
It feels wrong to remove davidx627 here. Should he be under `emeritus_approvers` (although he never has been an approver)?

Reply:
@pohly you can leave it commented out if you wish.

Reply:
@rajatgupta24 please do that.

Perhaps I am missing the purpose of `emeritus_approvers`, but why is that a field that the parser supports? Wouldn't comments serve the same goal (document former members)?

Reply:
@pohly I commented the section